main.tf

```diff
@@ -124,11 +131,28 @@ resource "google_container_node_pool" "main-node-pool" {
 
     oauth_scopes = [
       "compute-rw",
+      "storage-rw",
       "logging-write",
-      "monitoring"
+      "monitoring",
     ]
 
     service_account = "${google_service_account.compute.email}"
     tags            = ["k8s-node", "default-node-pool"]
   }
 }
+
+resource "google_storage_bucket" "benchmark-output-storage" {
+  name          = "${google_project.project.project_id}-benchmark-outputs"
+  location      = "${var.region}"
+  force_destroy = "true"
+  project       = "${google_project.project.project_id}"
+
+  retention_policy {
+    is_locked        = false
+    retention_period = 604800
```

Comment:
Why only keep for 7 days?

Reply:
So this was just an initial value to prevent accidental deletion. The bucket is not locked so this can be changed regardless. 

Got a value in mind?

Reply:
Have set this value to 365 days. So the bucket isn't locked meaning it can be deleted but should prevent accidental deletion of the objects.